apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -6313,6 +6350,10 @@ int tc_filesystem_main(int argc, char *argv[])
 #ifdef CONFIG_ITC_FS
 	itc_fs_main();
 #endif
+
+#ifdef CONFIG_AUTOMOUNT_USERFS
+	close(fd);
```

Comment:
Is there any reason to keep this fd until end of tc?
In my opinion, we could close it after getting the value immediately.